CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## v1.0.0-rc3 (unreleased)
 
-- No changes yet.
+- `Provide` and `Invoke` will now fail if `dig.In` or `dig.Out` structs
```

Comment:
Also it's _maybe breaking_ at best, if that's a thing. It only affects customers who have been misusing this in the first place and not getting those values in and out of the conatienr.